pkg/apis/integreatly/v1alpha1/installation_types.go

```diff
@@ -103,14 +115,15 @@ type PullSecretSpec struct {
 // InstallationStatus defines the observed state of Installation
 // +k8s:openapi-gen=true
 type InstallationStatus struct {
-	// INSERT ADDITIONAL STATUS FIELD - define observed state of cluster
+	// INSERT ADDITIONAL STATUS FIELDS - define observed state of cluster
 	// Important: Run "operator-sdk generate k8s" to regenerate code after modifying this file
 	// Add custom validation using kubebuilder tags: https://book.kubebuilder.io/beyond_basics/generating_crd.html
 	Stages             map[StageName]*InstallationStageStatus `json:"stages"`
 	PreflightStatus    PreflightStatus                        `json:"preflightStatus,omitempty"`
 	PreflightMessage   string                                 `json:"preflightMessage,omitempty"`
 	LastError          string                                 `json:"lastError"`
 	SetupGHCredentials bool                                   `json:"setupGHCredentials,omitempty"`
```

Comment:
@jjaferson I noticed that this is new :slightly_smiling_face: 

I think it would be nice if both this and the SMTP flag that I'm adding on the line below, were named consistently (at least I _think_ they have similar purposes). Would there be any issue with me changing this one to be `GitHubEnabled` or `GHCredentialsEnabled` or something like that (suggestions welcome)?

Reply:
not at all, I am not very creative when it comes to naming things, I'd go for `GHCredentialsEnabled` but any of them are good to me.

Reply:
Thanks! For now I've changed it to GitHubOAuthEnabled, since that's the thing being enabled.